ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -626,6 +626,29 @@ static void setTimeoutMax(RaftProperties properties, TimeDuration maxDuration) {
       setTimeDuration(properties::setTimeDuration, TIMEOUT_MAX_KEY, maxDuration);
     }
 
+    /** separate first timeout so that the startup unavailable time can be reduced */
+    String FIRST_TIMEOUT_MIN_KEY = PREFIX + ".first.timeout.min";
+    TimeDuration FIRST_TIMEOUT_MIN_DEFAULT = TimeDuration.valueOf(150, TimeUnit.MILLISECONDS);
+    static TimeDuration firstTimeoutMin(RaftProperties properties) {
+      final TimeDuration fallbackFirstTimeoutMin = Rpc.timeoutMin(properties);
```

Comment:
For fallback, we need to pass null for the log so that it won't print an additional log message.
See https://github.com/apache/ratis/blob/323bd1017afdbe03f70cccb172412e6fab221797/ratis-grpc/src/main/java/org/apache/ratis/grpc/GrpcConfigKeys.java#L103